sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -76,6 +76,32 @@ export class DisbursementOverawardService {
     return result;
   }
 
+  /**
+   * Adds a legacy overaward value to disbursement overawards table.
+   * @param studentId id of the student.
+   * @param overawardValue overaward value.
+   * @param disbursementValueCode disbursement value code.
+   * @param entityManager entity manager used to perform the query.
+   */
+  async addLegacyOveraward(
+    studentId: number,
+    overawardValue: number,
+    disbursementValueCode: string,
+    userId: number,
```

Comment:
As a recommendation, searching the code, most of the time the audit user is declared as below. It helps to differentiate from other moments where the `userId` is not provided with the audit intention.
```ts
* @param auditUserId user that should be considered the one that is causing the changes.
```

Reply:
Also I would suggest to use the name as auditUserId instead of userId.

Reply:
in the other [comment](https://github.com/bcgov/SIMS/pull/1739#discussion_r1116358148) I understood it should be 
any userId here because this method may be used to add legacy overaward using other users, not only audit users. Did I get it wrong?

Reply:
@andrepestana-aot the mentioned comment states to receive the user as a parameter and not use the "system user". On top of that name this user as `auditUserId` because it is a user and it is used for the sole intention of auditing.